src/studio/AppTemplates/AspNet/App/App.csproj

```diff
@@ -14,7 +14,7 @@
     <PackageReference Include="Altinn.App.Common" Version="3.2.3" />
     <PackageReference Include="Altinn.App.PlatformServices" Version="3.2.3" />
     <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="5.0.0" />
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.6.3" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.0.7" />
```

Comment:
quite a bump in version here.. but as it targets .net Standard 2.0 I guess it should be fine it the app still builds.

Reply:
There are two breaking changes, but I don't think they affect us here. Maybe take a closer look.

Reply:
I can't find that this breaking changes affect us.